### DISCUSSION TITLE
Automatic blog archive pages

### DISCUSSION THREAD

**Discussion by danielwelliott** [quarto-dev/quarto-cli#9651] [Q&A]
I have a WordPress blog that I'd like to migrate to a Quarto blog. I got everything installed and setup, migrated my posts (into a `site root/yyyy/mm` folder tree to maintain the same permalink structure as I had on WordPress), and the preview server has all the content with the url structure I want. However, archive pages for each year and month aren't generated and I can't find a way to automate this. I could manually add a page to each year and month folder with a listing for each, but that feels like something a static site generator should handle. Is it possible to have a Quarto website/blog create archive pages automatically for a permalink structure like blogsite.com/2024/05/my-great-post so that going to blogsite.com/2024/ lists all of the posts for 2024?

**Comment by mcanouil** [comment]
Create listings using glob pattern for each.
You could use a pre-render script to create those listings page by crawling your project.

There is also the custom listing approach such as https://www.andrewheiss.com/blog/
